webui/react/src/pages/ExperimentList.tsx

```diff
@@ -199,7 +200,7 @@ const ExperimentList: React.FC = () => {
           case Action.Activate:
             return setExperimentState({ experimentId: experiment.id, state: RunState.Active });
           case Action.Archive:
-            return archiveExperiment(experiment.id, true);
+            return archiveExperiment(experiment.id);
```

Comment:
separate: hmm we don't give them an option to unarchive here?

Reply:
we do, it's just a near the end of the switch block